be/src/udf/java/java_udf.cpp

```diff
@@ -162,7 +162,7 @@ void JVMFunctionHelper::_init() {
 
     name = JVMFunctionHelper::to_jni_class_name(UDAFStateList::clazz_name);
     jclass loaded_clazz = JNI_FIND_CLASS(name.c_str());
-    _function_states_clazz = std::make_unique<JVMClass>(std::move(loaded_clazz));
+    _function_states_clazz = new JVMClass(std::move(loaded_clazz));
```

Comment:
when will this object be released? Is it memory leak?

Reply:
@imay It will only be created once, along with the declaration of the end of the entire BE process.